pkg/controller/backup_controller.go

```diff
@@ -655,9 +656,17 @@ func (c *backupController) runBackup(backup *pkgbackup.Request) error {
 
 		//DataMover case
 		if datamover.DataMoverCase() {
-			err = datamover.WaitForDataMoverBackupToComplete(backup.Name)
+			err = datamover.CheckIfVSBFailed(backup.Name)
+
+			// first check for failed VSB
```

Comment:
Should we not just fail from plugin side ?

Reply:
If we only fail from the plugin side then the wait for VSBs to complete still runs and times out